packages/calling/src/Contacts/ContactsClient.ts

```diff
@@ -272,16 +273,15 @@ export class ContactsClient implements IContacts {
         const filteredContact = inputList.Resources.filter((item) => item.id === contactList[n])[0];
 
         const {displayName, emails, phoneNumbers, photos} = filteredContact;
-        const {sipAddresses} = filteredContact[SCIM_WEBEXIDENTITY_USER];
-        const firstName = filteredContact.name.givenName;
-        const lastName = filteredContact.name.familyName;
-        const manager = filteredContact[SCIM_ENTERPRISE_USER].manager.displayName;
-        const department = filteredContact[SCIM_ENTERPRISE_USER].department;
-
-        let avatarURL = '';
-        if (photos?.length) {
-          avatarURL = photos[0].value;
+        let sipAddresses;
+        if (filteredContact[SCIM_WEBEXIDENTITY_USER]) {
```

Comment:
Nit: Maybe we can use ternary operator here also.. but just a suggestion.

Reply:
@adhmenon the ternary operator doesn't work here. Something to do with the way typescript recognizes this object member. I initially tried with ternary but kept getting errors that SCIM_WEBEXIDENTITY_USER could be undefined.